crates/bevy_reflect/src/impls/std.rs

```diff
@@ -38,9 +38,9 @@ impl_reflect_value!(isize(Debug, Hash, PartialEq, Serialize, Deserialize));
 impl_reflect_value!(f32(Debug, PartialEq, Serialize, Deserialize));
 impl_reflect_value!(f64(Debug, PartialEq, Serialize, Deserialize));
 impl_reflect_value!(String(Debug, Hash, PartialEq, Serialize, Deserialize));
```

Comment:
Question: Should we specify custom type names for all these `std` types? Or continue to be okay with `alloc::string::String` and `core::time::Duration`?

Reply:
By default `impl_reflect_value!` use the name without the module path.
Because `module_path!` give the module path where `TypeName` is impl, so here `bevy_reflect::impls::std`.

Personnaly I think that std types must use a short name (without the module path).

Reply:
Yeah that makes sense to me. I’m inclined to agree.